lib/reporters/mini.js

```diff
@@ -216,7 +216,11 @@ MiniReporter.prototype.finish = function (runStatus) {
 	}
 
 	if (runStatus.failFastEnabled === true) {
-		status += '\n\n  ' + colors.failFast('`--fail-fast` is on. Any number of tests may have been skipped');
+		status += '\n\n  ' + colors.information('`--fail-fast` is on. Any number of tests may have been skipped');
+	}
+
+	if (runStatus.hasExclusive === true && runStatus.remainingCount > 0) {
+		status += '\n\n  ' + colors.information('The .only() modifier is used in some tests.', runStatus.remainingCount, 'test(s) were not run.');
```

Comment:
[Use `plur()`](https://github.com/avajs/ava/blob/23ffce31def7d960e9b39d38a95dbd3b5f103f81/lib/reporters/verbose.js#L79:L86), so "1 test was" and "2 tests were".